L1Trigger/CSCTriggerPrimitives/src/CSCGEMMatcher.cc

```diff
@@ -53,33 +55,31 @@ unsigned CSCGEMMatcher::calculateGEMCSCBending(const CSCCLCTDigi& clct, const GE
       else
         slope = lookupTableME21ILT_->es_diff_slope_L2_ME21_odd(diff);
     }
-  }
-
-  const bool isME1a(station_ == 1 and clct.getKeyStrip() > CSCConstants::MAX_HALF_STRIP_ME1B);
-
-  if (station_ == 1 and isME1a) {
-    if (isEven_) {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1a_even(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1a_even(diff);
-    } else {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1a_odd(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1a_odd(diff);
-    }
-  } else {
-    if (isEven_) {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1b_even(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1b_even(diff);
+  } else {  // station == 1
```

Comment:
```suggestion
  } else if( station_ == 1 ) {
```

Reply:
I suggest keeping this requirement explicit, since an erroneous user could in theory feed station 3 or 4 to the program without raising an assert. Whether we should catch that elsewhere is up for discussion.

Reply:
The CSCGEMMatcher is created for a CSCGEMMotherboard. Those are only made for station 1 or station 2, ring 1. It's unlikely, unless a mistake is made in the allocation of motherboards in the CSCTriggerPrimitivesBuilder class. Actually, it would be good to throw a logwarning if a CSCGEMMotherboard is made for stations other than ME1/1 or ME2/1...

Reply:
This could be added after L38 here: https://github.com/cms-sw/cmssw/blob/master/L1Trigger/CSCTriggerPrimitives/src/CSCGEMMotherboard.cc#L38

```c++
  // These LogErrors are sanity checks and should not be printed
  if (!isME11_) {
    edm::LogError("CSCGEMMotherboard") << "GEM-CSC OTMB constructed for a non ME1/1 chamber!";
  };

  if (!isME21_) {
    edm::LogError("CSCGEMMotherboard") << "GEM-CSC OTMB constructed for a non ME2/1 chamber!";
  };
```

Reply:
@dildik, if that is the very last change I can even integrate it in this PR.
But as I said, my intention was simply to fix a rather obvious bug in the code: if there are other changes planned, I think they should be rather left to the CSC/GEM experts. Please let me know what would you prefer.

Reply:
@perrotta If you prefer not to include it, that's fine. I can submit one.

Reply:
Thank you @dildick : yes, please do